enterprise/cypher/morsel-runtime/src/test/scala/org/neo4j/cypher/internal/runtime/vectorized/operators/MergeSortOperatorTest.scala

```diff
@@ -75,6 +94,30 @@ class MergeSortOperatorTest extends CypherFunSuite {
     out.longs should equal(Array(9, 9, 14,86,92))
   }
 
+  test("top on two morsels") {
```

Comment:
I don't think I'm seeing a test for `MergeSortOperator` targeting multiple sort columns or having additional columns. Would be nice to test.